irohad/main/application.cpp

```diff
@@ -324,6 +325,18 @@ Irohad::RunResult Irohad::restoreWsv() {
   };
 }
 
+Irohad::RunResult Irohad::validateKeypair() {
+  auto peers = storage->createPeerQuery() | [this](auto &&peer_query) {
+    return peer_query->getLedgerPeerByPublicKey(keypair.publicKey());
+  };
+  if (not peers) {
+    return expected::makeError(
+        "Failed to locate peer with provided public key!");
```

Comment:
From this message it is not clear, which public key is it about. Maybe rephrase like
```suggestion
        "There is no peer in the ledger with my public key!");
```

Reply:
Thanks, these suggestions sound good!